src/lucky/renderable.cr

```diff
@@ -96,6 +96,22 @@ module Lucky::Renderable
     end
   end
 
+  private def file(path,
```

Comment:
If you could add some inline documentation here that would be awesome. You can add `# :ditto:` to the method below to it is copied for that file too.

Reply:
So, I added some docs in there but the method in `Renderable` that delegates to the `FileResponse` class is private, so I guess it doesn't get generated to `./docs`. I put it, instead, inside the `FileResponse` class.  I think this is better actually as it gets closer to the implementation than a method that pretty much only delegates to it.

What do you think, @paulcsmith?

Reply:
Great idea! ❤️